Add script installation to second section in monitoring external resources guide (#1222)
sensu/sensu-docs: content/sensu-go/5.0/guides/monitor-external-resources.md

In this section of the guide, we'll use the [`proxy_requests` check attribute][3], along with [entity labels][11] and [token substitution][12], to monitor three sites using the same check.
Before we get started, go ahead and [install the HTTP check script][13] if you haven't already.

### Installing an HTTP check script

> First, we'll install a [bash script][4], named `http_check.sh`, to perform an HTTP

Reviewer: ```suggestion
If you haven't already, install a [bash script][4], named `http_check.sh`, to perform an HTTP
```
